package.json

```diff
@@ -54,6 +54,7 @@
     "react-dev-utils": "^12.0.0",
     "react-dom": "^17.0.2",
     "react-i18next": "^11.15.2",
+    "react-icons": "^4.3.1",
```

Comment:
We're already using FontAwesome Icons no need to add another library

Reply:
I agree, but I didn't know how to find and implement it and I thought it could be ok if I use react-icons library... I will fix it, master!